README.md

```diff
@@ -22,6 +22,13 @@ Read the full Parse Server guide here: https://github.com/ParsePlatform/parse-se
 * You now have a database named "dev" that contains your Parse data
 * Install ngrok and you can test with devices
 
+### Helpful Scripts:
+
+* `npm run lint` will check the linting of your cloud code and `index.js`.
```

Comment:
Do you want to mention where the setting are for lint?
> ... as defined in `.eslintrc.json`.

Reply:
And would it make sense to also add prettier as in Parse Server?

Reply:
I don't know, if you would like me to add prettier as well I am happy to!

Reply:
Up to you, adding lint is already an improvement from status quo.